javascript/house-style.md

```diff
@@ -195,6 +195,46 @@ We don't do this:
 const uniqueIDs = new Set([1, 2, 1, 2]);
 ```
 
+#### JS Doc
+
+Although your JavaScript code should be self-documenting, we highly recommend
+that you to document your JavasScript application or library using the
+Standardised [JS Doc](https://jsdoc.app/) format.
+
+This format is leveraged by the Language Server Protocol (LSP), which is
+supported in numerous editors and Integrated Development Environments (IDE)
+nowadays.  
+Thanks to LSP you get short lines of documentation right from the
+invocation context. This helps you code quickly, while avoiding obvious errors.
+
+For functions for instance, you get:
+- What it is intended to do
+- The type, default value, and description of its parameters
+- Which parameters are optional
+- The function returned value type and description
+
+Bonus: There are tools to generate a documentation website for your application
+or library if it uses JS Doc.
+
+A concrete and furnished example:
+
+```js
+/**
+ * Get a list of books from given a library.
```

Comment:
This reads a bit awkwardly.

Reply:
Thank you so much @danyalaytekin, I've committed all your great suggestions. I will work on making this one to read less awkwardly 🔎.

Reply:
Yeah thanks for the feedback Danyal! 🤗

Reply:
Thanks both 🙂 I need to have a good read through this playbook again as I keep seeing a lot of good stuff going in.  I don't have permissions to resolve this thread btw (in case you were waiting for me to)

Reply:
Hi @danyalaytekin, I've just pushed the hopefully improvement to the JSDoc example, in case you do not mind having another look at it. Thank you 🙏. I guess that I can resolve the conversion as soon as you feedback positively on it 🤷

Reply:
@morgaan https://github.com/springernature/frontend-playbook/pull/441/commits/684628849dd7a6c61b64c3038b00d4778f8760da looks great to me!